libraries/rust/common/src/btp_header.rs

```diff
@@ -144,7 +159,7 @@ mod tests {
     fn relay_bytes_to_signed_header() {
         let headers = load_test_headers();
         for header in headers {
-            let buff = hex::decode(header.encoded_protobuf.replace("0x", "")).unwrap();
+            let buff = hex::decode(header.encoded_protobuf.replace("", "")).unwrap();
```

Comment:
Can we remove the replace if it's not replacing anything?

Reply:
no its required. looks like pull from main patched it. CI has not covered these tests yet i guess.